README.md

```diff
@@ -3,7 +3,7 @@
 > Fast & simple storage. A Node.js-style `LevelDB` wrapper.
 
 [![level badge][level-badge]](https://github.com/level/awesome)
-[![npm](https://img.shields.io/npm/v/level.svg)](https://www.npmjs.com/package/level)
+[![Backers on Open Collective](https://opencollective.com/level/backers/badge.svg)](#backers) [![Sponsors on Open Collective](https://opencollective.com/level/sponsors/badge.svg)](#sponsors) [![npm](https://img.shields.io/npm/v/level.svg)](https://www.npmjs.com/package/level)
 ![Node version](https://img.shields.io/node/v/level.svg)
```

Comment:
Not sure what I think about the amount of badges we have now :)

![too-many-badges](https://user-images.githubusercontent.com/308049/35311902-48b5d7f4-00b9-11e8-8bdb-bb15bfa6da3b.png)

Reply:
I think it's fine 😁